src/js/node/fs.ts

```diff
@@ -836,6 +837,8 @@ function ReadStream(this: typeof ReadStream, pathOrFd, options) {
 
   // Get the stream controller
   // We need the pointer to the underlying stream controller for the NativeReadable
+  if (start && typeof start !== "number") throw ERR_INVALID_ARG_TYPE("options.start", "number", start);
+  if (end && typeof end !== "number") throw ERR_INVALID_ARG_TYPE("options.end", "number", end);
```

Comment:
Can you fix this in `.stream(offset)`? The code here isn't causing an assertion failure.